index.html

```diff
@@ -378,12 +425,16 @@ <h2 id="constraints">
           system.
         </p>
         <p>
-          Display capture sources MUST NOT be selected with the <code>deviceId</code> constraint,
-          since this would allow applications to influence selection. A display capture source is
-          represented in the <code>MediaStreamTrack</code> API as having a <code>deviceId</code>
-          parameter that is randomized each time a <code>MediaStreamTrack</code> is connected. The
-          only other constraint on this value is that it cannot duplicate any existing values for
-          <code>deviceId</code>.
+          Display capture sources cannot be selected with the <code>deviceId</code> constraint,
```

Comment:
It's not a replacement.  The previous paragraph establishes the requirement on implementers with a MUST NOT: you can't enumerate devices using `enumerateDevices`.  This is just explaining the consequences of that choice.